packages/dotlottie-js/src/common/dotlottie-common.ts

```diff
@@ -245,18 +246,33 @@ export class DotLottieCommon {
     });
   }
 
-  private _renameImageAssets(): void {
-    const images: Map<string, LottieImageCommon[]> = new Map();
+  /**
+   * Generates a map of duplicate image ids and their count.
+   * @returns Map of duplicate image ids and their count.
+   */
+  private _getMapOfDuplicateImageIds(): Map<string, number> {
+    let count = 0;
+    const dupeMap = new Map<string, number>();
 
     this.animations.forEach((animation) => {
-      images.set(animation.id, animation.imageAssets);
+      animation.imageAssets.forEach((imageAsset) => {
+        if (dupeMap.has(imageAsset.id)) {
```

Comment:
The `id` is not always the same as the `filename`, for example 

id: `image_0`, filename: `image_0.png`
is not the same as 
id: `image_0`, filename: 'image_0.webp'

right ? probably our ids should be the filename and we can get rid of using the image id in this case, no ?

Reply:
Inside the lottie itself, images had an id field, and then a name and path attribute, thats why I mapped it like this to the image class.

The id is just the filename without the extension
In your example the id stayed the same despite changing the extension